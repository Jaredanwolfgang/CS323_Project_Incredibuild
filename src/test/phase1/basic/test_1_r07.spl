int test_1_r07()
{
  int a, b, c;
  a = 1;
  b = a + 2; // original a && 2
  c = b - 3; // original b || 3
  a = c | 4;
  b = a & 5;
}
