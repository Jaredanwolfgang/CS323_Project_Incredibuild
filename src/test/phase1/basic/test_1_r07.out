Error type A at Line 7: Unknown lexeme |
Error type A at Line 8: Unknown lexeme &
